tests/components/deconz/test_sensor.py

```diff
@@ -174,6 +177,8 @@ async def test_sensors(hass):
     await gateway.async_reset()
 
     assert len(hass.states.async_all()) == 0
+    # Daylight sensor from deCONZ is added to set but is disabled by default
+    assert len(gateway.entities[sensor.DOMAIN]) == 1
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6239266)